.github/workflows/reminders_check.yml

```
@@ -14,4 +14,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check reminders and notify users
-        uses: agrc/reminder-action@v1
+        uses: agrc/reminder-action@v1.0.12
```

Hi @nessita  -- Previously actions have been pinned to a specific commit, for example https://github.com/django/django/commit/a464ead29db8bf6a27a5291cad9eb3f0f3f0472b

So maybe?

```suggestion
        uses: agrc/reminder-action@e59091b4e9705a6108120cb50823108df35b5392
```

---

Agreed. Tags can be changed, commit hashes are more robust.

---

Thank you both! I have pushed these changes in both actions.